java/src/main/java/com/google/oak/remote_attestation/InsecureAttestationVerifier.java

```diff
@@ -35,8 +37,9 @@ public class InsecureAttestationVerifier implements AttestationVerifier {
    * @return success value wrapped in a {@code Result}
    */
   @Override
-  public final Result<Boolean, Exception> verify(
-      final AttestationEvidence evidence, final AttestationEndorsement endorsement) {
-    return Result.success(true);
+  public final Result<AttestationResults, Exception> verify(long nowUtcMillis,
+      final Evidence evidence, final Endorsements endorsements,
+      final ReferenceValues referenceValues) {
+    return Result.success(AttestationResults.getDefaultInstance());
```

Comment:
I think it needs to return the correct server encryption public key. And this may require using an external Java library to parse the CBOR certificate.

Reply:
Yeah, you were right. Pulled in some external dependencies to handle CBOR and COSE